websites/D/Desu-Online/presence.ts

```diff
@@ -127,8 +127,14 @@ presence.on("UpdateData", async () => {
 			.querySelector("li.selected > a > div.playinfo > span")
 			.textContent.split("-");
 		presenceData.state = `Odcinek:${playinfo[0].replace("Odc", "")}`;
-		if (playinfo.length > 2)
-			presenceData.state += ` - "${playinfo[1].slice(1, -1)}"`;
+		if (playinfo.length > 2) {
+			let episodetitle = "";
+			for (let i = 0; i < playinfo.length; i++) {
+				if (i !== 0 && i !== playinfo.length - 1)
+					episodetitle += `${playinfo[i]}-`;
+			}
+			presenceData.state += ` - "${episodetitle.slice(1, -2)}"`;
+		}
```

Comment:
Couldn't you just do this:
`document.querySelector('[class="selected"]').querySelector("span").textContent.split(/- [0-9]/gm)[0]`

Reply:
i could but idk how regex works ;v
anyway ty

Reply:
i want text without `Odc 1 -`, only `Pies i piła mechaniczna`

Reply:
![image](https://user-images.githubusercontent.com/42370647/197257034-cf9afed0-67ad-4277-9207-7bc62c53c1da.png)

Reply:
then just change my suggested regex

Reply:
the thing is i don't really know how regex works

Reply:
nvm i found solution